gecoscc/templates/base_tree.jinja2

```diff
@@ -163,7 +145,8 @@
     <script src="{{request.application_url}}/static/js/libs/jquery.dataTables-1.9.4.min.js"></script>
     <script src="{{request.application_url}}/static/js/libs/DT_bootstrap.min.js"></script>
     <script src="{{request.application_url}}/static/js/libs/chosen.jquery-0.13.0.min.js"></script>
-    <script src="{{request.application_url}}/static/js/libs/jsonform-b3-1.0.0.js"></script>
```

Comment:
As we are changing the original jsonform & jsonpath, I suggest to change file names to something like jsonform-b3-1.0.0.gecos1.js & jsonpath-0.8.0.gecos1.js